src/server/src/main/java/io/cassandrareaper/service/RepairManager.java

```diff
@@ -218,6 +219,19 @@ public RepairRun startRepairRun(RepairRun runToBeStarted) throws ReaperException
     }
   }
 
+  public RepairRun editRepairRun(RepairRun repairRun, Double intensity) throws ReaperException {
```

Comment:
looking through `RepairManager` the method names are more explicit in their action names.
so i'm thinking here that `updateRepairRunIntensity(..)` would be a better name.
what do you think?

Reply:
I thought that this method can be extended in future by other parameters (not only intensity). That's why I named it in common case. According to open-closed principle you need only extend this method with new parameter if you want to edit something else. Without any renaming.

Reply:
let's do without premature design for extension for now. 

separate methods for separate actions seems to make more sense in this class for now.

Reply:
Okay